tests/test_onnx/test_head.py

```diff
@@ -0,0 +1,384 @@
+import pickle
+from functools import partial
+
+import mmcv
+import numpy as np
+import pytest
+import torch
+import torch.nn as nn
+
+from mmdet.models.dense_heads import RetinaHead, YOLOV3Head
+from .utils import get_data_path, list_gen, verify_model
+
+onnx_io = 'tmp.onnx'
+
+data_path = get_data_path()
+
+
+class WrapFunction(nn.Module):
+
+    def __init__(self, wrapped_function):
+        super(WrapFunction, self).__init__()
+        self.wrapped_function = wrapped_function
+
+    def forward(self, *args, **kwargs):
+        return self.wrapped_function(*args, **kwargs)
+
+
+def retinanet_config():
+    """RetinanNet Head Config."""
+
+    head_cfg = dict(
+        stacked_convs=6,
+        feat_channels=2,
+        anchor_generator=dict(
+            type='AnchorGenerator',
+            octave_base_scale=4,
+            scales_per_octave=3,
+            ratios=[0.5, 1.0, 2.0],
+            strides=[8, 16, 32, 64, 128]),
+        bbox_coder=dict(
+            type='DeltaXYWHBBoxCoder',
+            target_means=[.0, .0, .0, .0],
+            target_stds=[1.0, 1.0, 1.0, 1.0]))
+
+    test_cfg = mmcv.Config(
+        dict(
+            nms_pre=1000,
+            min_bbox_size=0,
+            score_thr=0.05,
+            nms=dict(type='nms', iou_threshold=0.5),
+            max_per_img=100))
+
+    model = RetinaHead(
+        num_classes=4, in_channels=1, test_cfg=test_cfg, **head_cfg)
+    model.requires_grad_(False)
+    model.eval()
+
+    return model
+
+
+def yolo_config():
+    """YoloV3 Head Config."""
+
+    head_cfg = dict(
+        anchor_generator=dict(
+            type='YOLOAnchorGenerator',
+            base_sizes=[[(116, 90), (156, 198), (373, 326)],
+                        [(30, 61), (62, 45), (59, 119)],
+                        [(10, 13), (16, 30), (33, 23)]],
+            strides=[32, 16, 8]),
+        bbox_coder=dict(type='YOLOBBoxCoder'))
+
+    test_cfg = mmcv.Config(
+        dict(
+            nms_pre=1000,
+            min_bbox_size=0,
+            score_thr=0.05,
+            conf_thr=0.005,
+            nms=dict(type='nms', iou_threshold=0.45),
+            max_per_img=100))
+
+    model = YOLOV3Head(
+        num_classes=4,
+        in_channels=[1, 1, 1],
+        out_channels=[16, 8, 4],
+        test_cfg=test_cfg,
+        **head_cfg)
+    model.requires_grad_(False)
+    model.eval()
+
+    return model
+
+
+@pytest.mark.skipif(
```

Comment:
If all test in this file require torch >=1.5.0, you could skip the whole test module by adding  a line like below at the beginning of the file:
```
if not torch.cuda.is_available():
    pytest.skip(
        'CUDA is required for this test module', allow_module_level=True)
```